composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=5.4",
-    "moxie-lean/wp-endpoint": "1.*.*",
+    "moxie-lean/wp-endpoint": "2.*.*",
```

Comment:
OK, I see. But I think this caused issues because of the way the parent dependencies are set up (also using the * format). It would be best to leave it like this for consistency and look at fixing all our dependency formats in one go.